### DISCUSSION TITLE
Getting started with the Streaming API in an Andorid project

### DISCUSSION THREAD

**Discussion by MohamedElshihawy** [deepgram/community#662] [General help]
It would be helpful if there are any resources to get started with the streaming API through socket connection in android
.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
